python/setup.py

```diff
@@ -2,7 +2,7 @@
 setup(
     name = 'pytispark',
     packages = ['pytispark'],
-    version = '1.1',
+    version = '1.2.1',
```

Comment:
if you update pytispark version, it should be released on pyti

Reply:
shall we release pytispark?

Reply:
we may need to release pytispark when changes involve in logic or spark version.

Reply:
reverted